### DISCUSSION TITLE
2023年IDEA优秀设计2 - 晦涩弗里曼的博客

### DISCUSSION THREAD

**Discussion by giscus** [obscurefreeman/blog#9] [Giscus]
# 2023年IDEA优秀设计2 - 晦涩弗里曼的博客

这次的IDEA获奖作品有什么亮点？

https://obscurefreeman.github.io/blog/%E8%AE%BE%E8%AE%A1/IDEA%E4%BC%98%E7%A7%80%E8%AE%BE%E8%AE%A12/

**Comment by obscurefreeman** [comment]
非常前卫的设计理念。
